src/explore-education-statistics-common/src/modules/find-statistics/components/DataBlock.tsx

```diff
@@ -175,7 +173,13 @@ class DataBlock extends Component<DataBlockProps, DataBlockState> {
                 {tables.map((table, idx) => {
                   const key = `${id}0_table_${idx}`;
 
-                  return <TimePeriodDataTableRenderer key={key} {...table} />;
+                  return (
+                    <TimePeriodDataTableRenderer
+                      heading={heading}
+                      key={key}
+                      {...table}
```

Comment:
Where you put the object spread matters. In this context, it seems like it would need to be before the other props.